content/300-guides/200-deployment/110-deployment-guides/300-deploying-to-vercel.mdx

```diff
@@ -1,165 +1,320 @@
+You have successfully deployed the app to Vercel. Note that Vercel will build and deploy further commits to your GitHub repository.
 
-## <inlinecode>vercel/pkg</inlinecode> <inlinecode>ENOENT</inlinecode> error
+To check that the API returns data successfully, click on the image of the deployed application. This will take you to your application at `https://__PROJECT-NAME__.vercel.app/`, where `__PROJECT-NAME__` is the name that Vercel generates for the project.
 
-If you use [vercel/pkg](https://github.com/vercel/pkg) to package your Node.js project, then you might encounter an error like the following:
+When the page loads, click **Check API status**. If successful, you will see a result similar to the image below.
 
-```
-spawn /snapshot/enoent-problem/node_modules/.prisma/client/query-engine-debian-openssl-1.1.x ENOENT
+```code no-copy
+{"up":true}
 ```
 
-To avoid this error, add your Prisma query engine binary path to the `pkg/assets` section of your `package.json` file, as follows:
+## 2. Test your deployed application
 
-```json file=package.json copy
-{
-  "pkg": {
-    "assets": ["node_modules/.prisma/client/*.node"]
-  }
-}
-```
+You can now test your application. Open your application at `https://__PROJECT_NAME__.vercel.app` to interact with the API.
+
+### Steps
+
+1. Click **Load users with profiles**.
+
+   You do not yet have any data in the database, so the application returns an empty array:
+
+   ```code no-copy
+   []
+   ```
+
+   <Admonition type="info">
+
+   This API call uses Prisma to get all `users` in the database with their related `profiles`. The code is in the `pages/api/users.js` file in the project repository.
+
+   </Admonition>
+
+2. Click **Seed data**.
+
+   <Admonition type="info">
+
+   This API call uses Prisma to delete any existing database records and load the database with test `users`, `profiles`, and `posts`, then returns the created `users`. The code is in the `pages/api/seed.js` file.
+
+   </Admonition>
+
+3. Click **Load users with profiles** again.
+
+   This now returns the `users` and `profiles` that you added in the previous step:
+
+   ```code no-copy
+   [
+   {
+   "id": 5,
+   "email": "jane@prisma.io",
+   "name": "Jane",
+   "profiles": [
+     {
+       "id": 13,
+       "bio": "Technical Writer",
+       "userId": 5
+     },
+     {
+       "id": 14,
+       "bio": "Health Enthusiast",
+       "userId": 5
+     },
+     {
+       "id": 15,
+       "bio": "Self Quantifier",
+       "userId": 5
+     }
+   ]
+   },
+   {
+   "id": 6,
+   "email": "toru@prisma.io",
+   "name": "Toru Takemitsu",
+   "profiles": [
+     {
+       "id": 16,
+       "bio": "Composer",
+       "userId": 6
+     },
+     {
+       "id": 17,
+       "bio": "Musician",
+       "userId": 6
+     },
+     {
+       "id": 18,
+       "bio": "Writer",
+       "userId": 6
+     }
+   ]
+   }
+   ]
+   ```
```

Comment:
Indentation is off in these code blocks. This should be:

````
   ```code no-copy
   [
     {
       "id": 5,
       "email": "jane@prisma.io",
       "name": "Jane",
       "profiles": [
         {
          "id": 13,
          "bio": "Technical Writer",
          "userId": 5
         },
         {
          "id": 14,
          "bio": "Health Enthusiast",
          "userId": 5
         },
         {
          "id": 15,
          "bio": "Self Quantifier",
          "userId": 5
         }
       ]
     },
     {
       "id": 6,
       "email": "toru@prisma.io",
       "name": "Toru Takemitsu",
       "profiles": [
         {
          "id": 16,
          "bio": "Composer",
          "userId": 6
         },
         {
          "id": 17,
          "bio": "Musician",
          "userId": 6
         },
         {
          "id": 18,
          "bio": "Writer",
          "userId": 6
         }
       ]
     }
   ]
   ```
````